### PR TITLE
fix: netpol allow worker internet

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Fixes FL-1611

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: charts/substra-backend/templates/statefulset-worker.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ spec:
         app.kubernetes.io/component: substra-worker
         role-api-k8s-client: 'true'
         role-database-client: 'true'
+        # Needed for downloading models from other backends + connecting to remote registry
+        role-internet-egress: 'true'
```
Comment (thbcmlowk) [inline charts/substra-backend/templates/statefulset-worker.yaml:39]: Do we have leads on why this was working both with skaffold and on substra-dev but not on staging?

Comment (guilhem-barthes) [reply charts/substra-backend/templates/statefulset-worker.yaml:39]: Was the post-mortem not clear enough?

**Review by SdgJlbl — APPROVED** [review]
🕸️

File: charts/substra-backend/templates/statefulset-worker.yaml (hunk 1 above)
Comment (thbcmlowk) [reply charts/substra-backend/templates/statefulset-worker.yaml:39]: Very clear, thank you. Missed the thought process of it somehow when I first read it.

**Pull request merged**